### PR TITLE
add-even

### PR DESCRIPTION
**Pull request by pako2200**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: add-even
🏅 Granted 5 XP for @pako2200

**Comment by ziobron** [comment]
⏱ 3 XP extra for @pako2200 for on-time delivery

File: homework/add-even/addEven.hpp [hunk 1]
```diff
@@ -1,8 +1,14 @@
 #pragma once
 #include <vector>
 
-int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+int addEven(const std::vector<int> &numbers)
+{
+
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:6]: Redundant new line

File: homework/add-even/addEven.hpp [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+    int sum_numbers{0};
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:7]: add space

File: homework/add-even/addEven.hpp [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
+
+    for (int number : numbers)
+    {
+        sum_numbers += number % 2 == 0 ? number : 0;
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:11]: Better use if instead add `0`

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**